apps/docs/content/components/CTAForm/index.mdx

```diff
@@ -0,0 +1,101 @@
+
+## Anatomy
+
+![An image displaying a the CTA Form anatomy, with a label on top of an input form, followed by a CTA button and optional checkbox underneath the input.](https://github.com/primer/brand/assets/131988618/2c86dd18-f3b0-41e3-a0d3-cca004ea37b1)
+
+- Label: Short text that describes the required input form. 
+- Input: Form requesting information. 
+- Action: Call-to-action button that corresponds after completing the input form. 
+- Checkbox: An optional checkbox to request additional behavior.
+
+## Usage
+
+The CTA form is designed to capture user input and call to actions on a website. It serves as an efficient tool to prompt user engagement and action. While it can be placed throughout the page, it is encouraged to be positioned at the bottom to conclude the user journey, summarize key information, and encourage users to take action.
+
+The CTA form presents a concise label, prompting users to provide specific input in the form field. Once users enter the required information, they can trigger the call-to-action through the button. Additionally, the CTA form supports an optional checkbox to seek additional user behavior or acknowledge certain terms.
+
+By adopting the same guidelines as the respective [form](/components/TextInput) component, [button](/components/Button) component, and [checkbox](/components/Checkbox) component, the CTA form ensures consistency in design and enhances user familiarity with the interface.
+
+<DoDontContainer>
+  <Do>
+<img src="https://github.com/primer/brand/assets/131988618/561b62c9-c328-4c7e-98e5-18e5a5d7ad42" />
+    <Caption>Place the CTA Form near the end of the page to conclude the user journey and encourage users to take action.</Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/131988618/0cb06dc0-b885-4b4d-a997-cd42da72f798" />
+    <Caption>
+      Don't place the CTA Form early on in the user journey. Use the Hero or CTA Banner component instead.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+Keep the CTA Form simple and intuitive to minimize friction and encourage user engagement. Ensure there is enough context surrounding the component. Utilize the label as a description of the form input. 
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/131988618/15cc3be6-89cb-4b63-ab03-9236693f66c3" />
+    <Caption>
+      Include context in areas surrounding the CTA Form to encourage user engagement. 
+    </Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/131988618/2333fa03-4bd3-488d-ae41-81e3511259ca" />
+    <Caption>
+        Don't use the CTA Form without context. The label should be used to describe the input, not to provide context.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+## Options
+
+### Size
+
+The CTA Form includes a medium and a small size option. Choose the sizing of the CTA Form to best ensure consistency in design and enhance user familiarity with the interface.
```

Comment:
Hi Josep 👋 ! 
![image](https://github.com/primer/brand/assets/131988618/30b0b4cb-d74b-4587-a90c-b75d6e545789)
I saw this example in the primer brand figma. Sounds great! If there are two options (med/small), should we keep it in size?

Reply:
Hey @jesskuo4 👋 ! the `/Small` and `/Medium` options represents the viewport size in case consumers needs to design for mobile, tablet or large screens in Figma. Some components changes their layout depending the screen width and we opted to have different components as Figma doesn't provide a clean way to handle them. 

In short, the component doesn't provide options to change the layout, it happens out of the box depending the viewport size.


You can see how the component changes the layout and behaves different by resizing the screen in the storybook examples: https://primer.style/brand/storybook/?path=/story/components-ctaform--default